### DISCUSSION TITLE
Is MSYS2 (a Linux chain tool emulator for Windows) supported ? https://www.msys2.org/

### DISCUSSION THREAD

**Discussion by serge-hulne** [vlang/v#17292] [Installation Issues]
Is MSYS2 (a Linux chain tool emulator for Windows :  `https://www.msys2.org` ) supported ? 

I tried to compile V from source under MSYS2, but it doesn't seem to be supported.

Did anybody else try?

**Comment by JalonSolov** [comment]
Better off installing gcc/clang from https://winlibs.com - many of us have used it and it works fine.

**Reply by getfile** [reply]
have package manager in winlibs?

**Reply by JalonSolov** [reply]
No, it's just an archive.  Extract it where you wish, and add the `bin` subdir to your path.

**Comment by serge-hulne** [comment]
Does WinLibs provide something that MingW does not ?

**Reply by JalonSolov** [reply]
It provides natively compiled binaries of the tools, instead of tools which must rely on dlls, etc.  You don't need any of the MSYS ecosystem with the winlibs versions.  The winlibs versions are also kept up with the actual latest versions available on Linux.

**Comment by serge-hulne** [comment]
I will give it a try.
